frontend/src/components/editor-page/app-bar/cheatsheet/use-components-from-app-extensions.tsx

```diff
@@ -3,7 +3,7 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-import { optionalAppExtensions } from '../../../../extensions/extra-integrations/optional-app-extensions'
+import { allAppExtensions } from '../../../../extensions/extra-integrations/all-app-extensions'
```

Comment:
I'm still not 100% convinced with the naming when I look at the path. Especially the mix of "integration" and "extension". Another point is that "all" is located below "extra"...

Reply:
I reorganized everything.